h_w__12/args_f.py

```diff
@@ -0,0 +1,12 @@
+def elem(*args):
+    my_elements = sorted(*args)
+    print(f' Minimum element: {my_elements[0]}, maximum element: {my_elements[-1]}')
```

Comment:
It should be 2 separate functions for min and max. And the function should return smth, not print